storage/storage_dest.go

```diff
@@ -555,20 +594,26 @@ func (s *storageImageDestination) queueOrCommit(index int, info addedLayerInfo)
 	return nil
 }
 
-// getDiffIDOrTOCDigest returns the diffID for the specified digest or the digest for the TOC, if known.
-func (s *storageImageDestination) getDiffIDOrTOCDigest(uncompressedDigest digest.Digest) (digest.Digest, bool) {
+// getLayerID returns the diffID for the specified digest or the digest for the TOC, if known.
+func (s *storageImageDestination) getLayerID(uncompressedDigest digest.Digest, tocDigest digest.Digest, index int) (string, bool) {
```

Comment:
(This might accept an `addedLayerInfo` as a single parameter.)

Reply:
In #2288, only one field of `addedLayerInfo` is necessary, removing the benefit of this change.